platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/spi/info/NetworkInfo.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.spi.info;
+package com.swirlds.state.spi.info;
```

Comment:
This is not related to States API, Should this be moved? Are we moving everything under spi to `com.swirlds.state.` ?

Reply:
I wish it wasn't related to States API but it is. Though indirectly 

![image](https://github.com/hashgraph/hedera-services/assets/1595361/2ce1a59d-d866-4d96-b87b-4349742d885c)

In the initial implementation of this PR I decided to move this interface to `swirlds-platform-core` as currently contains the implementation details related to the state. 

There is a follow up ticket for this change: 
https://github.com/hashgraph/hedera-services/issues/13270

You intuition is correct, and `swirlds-state-api` is not the best location for these classes but it's still better than `swirlds-platform-sdk`. I didn't want to include this change in the current PR because it's already huge.